### PR TITLE
feat(denom): update to stATOM

### PR DESCRIPTION
**Pull request by 0xpatrickdev**
### What was added 
- [x] change `STATOM1` to `STATOM`, and use `ibc/49C630713B2AB60653F76C0C58D43C2A64956803B4D422CACB6DD4AD016ED846` denom
- [x] uses `stATOM` for displayName. The asset `keyword` ~~and `brand`~~ must begin with an uppercase letter, so `STATOM` is used

### CODE REVIEW

**Comment by 0xpatrickdev** [comment]
@dckc I can't tag you as a reviewer until you join as a collaborator (invite just sent). Feel free to comment in lieue of approval!

**Comment by dckc** [comment]
> The asset keyword and brand must begin with an uppercase letter, so STATOM is used

strictly speaking, only keywords have to start with uppercase.
But somewhere in this code, we're lazy and re-use a brand name for a keyword. sigh

_for follow-up, see #16_

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ make fund-provision-pool
 # create a smart wallet
 ACCT_ADDR=your-address make provision-acct
 # add faucet tokens to wallet
-ACCT_ADDR=your-address FUNDS=80000000ubld,80000000uist,80000000ibc/toyatom,80000000ibc/toystatom1 make fund-acct
+ACCT_ADDR=your-address FUNDS=80000000ubld,80000000uist,80000000ibc/toyatom,80000000ibc/toystatom make fund-acct
```
Comment (dckc) [inline README.md:38]: this change isn't strictly necessary. but it's fine.

**Review by dckc — APPROVED** [review]
looks good

**Pull request merged**